pkgs/applications/office/wikidpad/2.3.nix

```diff
@@ -0,0 +1,38 @@
+#References
+# http://trac.wikidpad2.webfactional.com/wiki/InstallLinux
+
+{stdenv, lib, fetchFromGitHub, python27, makeWrapper, wrapGAppsHook}:
+let
+  fetchGitHubJSON = file: fetchFromGitHub { inherit (lib.importJSON file) owner repo rev sha256; };
```

Comment:
To me, it seems excessive to have this JSON file and all this extra machinery to fetch the source when we are only getting one revision. Can we make this just be a normal fetchFromGitHub, like most of nixpkgs?